pkgs/tools/networking/p2p/tahoe-lafs/default.nix

```diff
@@ -60,7 +60,8 @@ python3Packages.buildPythonApplication rec {
     html5lib magic-wormhole netifaces pyasn1 pycrypto pyutil pyyaml recommonmark
     service-identity simplejson sphinx_rtd_theme testtools treq twisted zfec
     zope_interface
-  ];
+  ] ++ twisted.extras-require.tls
+    ++ twisted.extras-require.conch;
```

Comment:
fwiw I have been trying to adopt this package but I can't get a review on https://github.com/NixOS/nixpkgs/pull/166601/files